test/mock_data/get_users_2.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
As discussed personally, i agree with you in this case. And for further mock responses i'l think of a better way, like templating